pages/community/[title].tsx

```diff
@@ -12,7 +12,7 @@ import {
 } from '../../components/trending/Trending.styles';
 import request from '../../modules/request';
 import { CommunityType } from '../../modules/types';
-import { AppWrapper, Content } from '../../styles/AddCommunity.styles';
+import { AppWrapper, Content, Div } from '../../styles/AddCommunity.styles';
```

Comment:
This is something I didnt notice before, but we should never import styles from another page. This should have it's own "community.styles.ts" file and import from there. If some layout components like AppWrapper and Content are common, then probably we should move them to become actual components (in the components folder). Let's discuss tomorrow :)